src/main/java/org/jboss/xavier/integrations/route/MainRouteBuilder.java

```diff
@@ -148,23 +174,23 @@ public void configure() {
 
         from("direct:calculate-costsavings")
                 .id("calculate-costsavings")
-                .doTry()
-                    .transform().method("calculator", "calculate(${body}, ${header.MA_metadata})")
-                    .log("Message to send to AMQ : ${body}")
-                    .to("jms:queue:uploadFormInputDataModel")
-                .endDoTry()
-                .doCatch(Exception.class)
-                    .to("log:error?showCaughtException=true&showStackTrace=true")
-                    .setBody(simple("Exception on parsing Cloudforms file"))
+                .transform().method("calculator", "calculate(${body}, ${header.MA_metadata})")
+                .log("Message to send to AMQ : ${body}")
+                .to("jms:queue:uploadFormInputDataModel")
                 .end();
     }
 
+    private Predicate isResponseSuccess() {
+        return header(Exchange.HTTP_RESPONSE_CODE).isEqualTo(HttpStatus.SC_OK);
```

Comment:
I suppose this implementation covers the remote services we call right now and maybe, to avoid changes in the future, it's worth adding other `2XX` codes as well in this method.
For example i feel like it's worth adding `201` (`SC_CREATED`), `202` (`SC_ACCEPTED`) and `204` (`SC_NO_CONTENT`).

Reply:
That's why I created this method.....we can have a list of Success codes that we could configure in the yaml for instance.
But, I will add these other you mention : 201, 202
are you sure about 204 ?

Reply:
I added `204` because we use it so i hope other will use it as well :wink: